internal/post/sender.go

```diff
@@ -76,7 +76,12 @@ func (sender *SMTPSender) SendEmail(msg *Message) error {
 		err = errs.Combine(err, data.Close())
 	}()
 
-	_, err = data.Write(msg.Bytes())
+	mess, err := msg.Bytes()
+	if err != nil {
+		return err
```

Comment:
This should be `return Error.Wrap(err)` however all the errors of this function are returned as they so I would suggest to leave it as it's now and open an issue in Jira to revisit all the methods and functions of this package to check the ones that should be to be `Error.Wrap(..)` which they will probably be the most of them